integration_tests/test_pip_import_01.py

```diff
@@ -0,0 +1,39 @@
+from lpynn.perceptron import init_perceptron, print_perceptron, normalize_input_vectors, Perceptron, train_dataset
+from lptypes import i32, f64
```

Comment:
The added test is such that both the packages it uses/needs are not available in the source code of `lpython`. (They are present in the `lpython` repository with the names `lnn` and `lpython`.) The above test makes sure that we are not using packages from any `lpython` project sub-directory and instead these are being used from the `pip` installed packages.

Reply:
The `lpynn` is fine. I would be careful with `lptypes`. I don't have an opinion right now, but I would be conservative and not allow it for now, and require that the special types that LPython understands are imported from "lpython" only, that is our own blessed version.